app/models/Commodity.scala

```diff
@@ -30,10 +28,18 @@ case class Commodity(
   validityEndDate: Option[Instant]
 ) {
 
```

Comment:
Reason for the below changes here is I noticed the commodity isValid tests would fail when passed a Instant.now() - to ensure the instants are comparable I've made it so we remove the time as I believe we only go off the date any ways